### ISSUE TITLE
System.Net.Http causes error CS0433

### ISSUE THREAD

**Issue by Suyogya** [dotnet/corefx#16993]
When referencing System.Net.Http in .net standard library 1.6 app. I get following compile error

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0433	The type 'FormUrlEncodedContent' exists in both 'System.Net.Http, Version=1.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' and 'System.Net.Http, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'

**Comment by karelz** [comment]
Do you have a small repro you could share?
@davidsh any idea? cc @weshaggard @ericstj

**Comment by davidsh** [comment]
The error means it type is being defined in two different dlls with the same name but different versions.
https://msdn.microsoft.com/en-us/library/64wh5743.aspx

This dll, 'System.Net.Http, Version=1.5.0.0' is not part of .NET core. I don't know where it comes from.

You will need to attach a full Visual Studio solution in order to figure this out.  It is coming from an unsupported NuGet package of some kind.

**Comment by Suyogya** [comment]
Thank you for the prompt reply. I looked at the references again. It looks like I was including 
`Microsoft.AspNet.WebApi.Client` in the project and that was causing the issue. Thanks again.

**Comment by dagrooms52** [comment]
I have the same issue, but I actually need to keep `Microsoft.AspNet.WebApi` in my project. Any fix other than waiting for 2.0.0?

**Comment by karelz** [comment]
Can you please check with `System.Net.Http, Version=1.5.0.0` producers? (I guess they are also authors of `Microsoft.AspNet`)
